build.ps1

```diff
@@ -19,23 +19,32 @@ param (
     [switch]$Clean,
     # When specified, cargo test will be invoked for the specified component.
     [switch]$Test,
+    # When specified, cargo clippy will be invoked for the specified component.
+    [switch]$Lint,
+    # Run rustfmt
+    [switch]$Fmt,
     # When specified a build will not be run.
     [switch]$SkipBuild,
     # Use a optimized release build
     [switch]$Release,
     # Run a cargo check instead of build
     [switch]$Check,
-    # Add protocol feature
-    [switch]$protocols
+    # Features to pass to cargo
+    [string]$Features,
+    # Options to pass to the cargo test command
+    [string]$TestOptions,
+    # Use nightly toolchain
+    [switch]$Nightly
```

Comment:
Minor, but would there be benefit to making a `-Toolchain` option instead to allow you to set arbitrary toolchains?

(I'm really just thinking out loud... what's here now is certainly fine, and it's not like it'd be difficult to change in the future if we needed to.)